src/common/util/src/file_util.cpp

```diff
@@ -87,6 +87,25 @@ std::string ov::util::get_directory(const std::string& s) {
     return rc;
 }
 
+#ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
+std::wstring ov::util::get_directory(const std::wstring& s) {
+    std::wstring rc = s;
+    // Linux-style separator
+    auto pos = s.find_last_of('/');
```

Comment:
Can we use `FileTraits:: file_separator `?

Reply:
Ok, I amended it.